pkg/dotc1z/file.go

```diff
@@ -12,7 +13,8 @@ import (
 )
 
 func loadC1z(filePath string) (string, error) {
-	workingDir, err := os.MkdirTemp("", "c1z")
+	tmpDir := path.Dir(filePath)
+	workingDir, err := os.MkdirTemp(tmpDir, "c1z")
```

Comment:
there isn't a great way to flow this through without breaking some APIs. Most of the usages of this are already in a tempdir using the correct defined path, so we use the same directory and create a new tempdir there to use. This dir is removed on cleanup.